cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -259,3 +262,13 @@ type HostPathMount struct {
 type KubeProxy struct {
 	Config *kubeproxyconfigv1alpha1.KubeProxyConfiguration
 }
+
+// AuditPolicyConfiguration holds the options for configuring the api server audit policy.
+type AuditPolicyConfiguration struct {
+	// Path is the local path to an audit policy.
+	Path string
+	// LogDir is the local pth to the directory where logs should be stored.
+	LogDir string
+	// LogMaxAge is the number of days logs will be stored for. 0 indicates forever.
+	LogMaxAge *int
+}
```

Comment:
Add the same TODO here? i.e., `//TODO(chuckha) add other options for audit policy`